docs/application/native/guides/app-management/sticker.md

```diff
@@ -0,0 +1,427 @@
+  {
+      int ret;
+      ret = sticker_provider_insert_data(sticker_provider, sticker_data);
+      if (ret != STICKER_ERROR_NONE)
+          /* Error handling */
+  }
+  ```
+
+- To store multiple sticker information, use `sticker_provider_insert_data_by_json_file()`:
+
+  ```
+  void
+  insert_finished_cb(sticker_error_e error, void *user_data)
+  {
+      /* Your code */
+  }
+
+  void
+  insert_sticker_data(sticker_provider_h sticker_provider, const char* json_path)
+  {
+      int ret;
+      ret = sticker_provider_insert_data_by_json_file(sticker_provider, json_path, insert_finished_cb, NULL);
+      if (ret != STICKER_ERROR_NONE)
+          /* Error handling */
+  }
+  ```
+
+- To store sticker information without using `sticker_provider_insert_data()` and `sticker_provider_insert_data_by_json_file()`, add the following metadata to the `tizen-manifest.xml` file:
+
+  ```
+  <category name="http://tizen.org/category/sticker"/>
+  <!-- Sets the JSON local path -->
+  <metadata key="http://tizen.org/metadata/sticker" value="/res/json/sticker.json"/>
+  ```
+
+  If you set the path of the JSON file to the value in metadata, sticker information of the JSON file is automatically stored when the Sticker provider application is installed.
+
+  > **Note**
+  >
+  > The metadata must be added after category.
+
+### Updating Sticker Information
+
+You can update the sticker information stored in the Sticker database using the following functions:
+
+```
+void
+update_sticker_data(sticker_provider_h sticker_provider, sticker_data_h sticker_data)
+{
+    int ret;
+    ret = sticker_provider_update_data(sticker_provider, sticker_data);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+### Destroying Sticker Provider Handle
+
+Destroy the Sticker provider handle using `sticker_provider_destroy()`:
+
+```
+void
+destroy_sticker_provider_handle(sticker_provider_h sticker_provider)
+{
+    int ret;
+    ret = sticker_provider_destroy(sticker_provider);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+<a name="consumer_application"></a>
+## Creating Sticker Consumer Application
+
+You can create the Sticker consumer application using Sticker API to use sticker information. You can search for stickers and get details of stickers.
+
+### Creating Sticker Consumer Handle
+
+Create the Sticker consumer handle using `sticker_consumer_create()` that allows you to retrieve the sticker information:
+
+```
+void
+create_sticker_consumer_handle()
+{
+    sticker_consumer_h sticker_consumer;
+    int ret;
+    ret = sticker_consumer_create(&sticker_consumer);
+    if (ret != STICKER_ERROR_NONE)
+        /* Error handling */
+}
+```
+
+<a name="retrieve_sticker"></a>
+### Retrieving Sticker Information
+
+You can retrieve all the stickers stored in the Sticker database and use keyword, group name, and type of URI to retrieve stickers.
+
+The following examples show how to retrieve stickers using the group name:
```

Comment:
Please rephrase to "example shows":

The following example shows how to retrieve stickers using the group name:

Reply:
Fixed.